File: src/com/test/kotlin/pojo/TestPOJO.java
package com.test.kotlin.pojo;

import java.util.Arrays;
import java.util.List;

public class TestPOJO {

	public static void main(String[] args) {
		 
		List<String> childNames = Arrays.asList("child1", "child2");
				
		Employee obj = new Employee("abcd", 1, true, childNames);
	        
		System.out.println(obj);
		
		// output: abcd, 1, true, [child1, child2]
		// refer toString method

	}

}

File: src/com/test/kotlin/pojo/TestPOJO.kt
package com.test.kotlin.pojo

fun main(args: Array<String>) {
	
	var childNames: List<String> = listOf("child1", "child2")

    val obj:Employee = Employee("abcd", 1, true, childNames)
        
	println(obj)
	
	// output: abcd, 1, true, [child1, child2]
	// refer toString method
}